packages/cosmic-swingset/Makefile

```diff
@@ -178,19 +187,13 @@ show-local-gci:
 
 set-local-gci-ingress:
 	set -e; \
-	gci=`./calc-gci.js ~/.ag-chain-cosmos/config/genesis.json`; \
-	rpcport=`./calc-rpcport.js ~/.ag-chain-cosmos/config/config.toml`; \
-	for dir in t1/*; do \
+	gci=`./calc-gci.js t1/n0/config/genesis.json`; \
```

Comment:
does this mean we no longer touch `~/.ag-chain-cosmos` while running scenario2? awesome.